apps/backend/core/configuration/validation/utils/__init__.py

```
@@ -0,0 +1,26 @@
+    check_api_key,
+    validate_http_endpoint,
+)
+
+__all__ = [
+    # HTTP utilities
+    "validate_http_endpoint",
+    "check_api_key",
+    # Database utilities
+    "validate_postgres",
+    "validate_redis",
+]
```

Ruff (0.14.4)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)